lib/common/xml.c

```diff
@@ -516,6 +516,14 @@ pcmk__xe_match(xmlNode *parent, const char *node_name,
     return NULL;
 }
 
+/*!
+ * \brief Copy properties from source XML to target XML
+ *
+ * \param[in]  target  XML to copy properties to
+ * \param[in]  src     XML to copy properties from
+ *
+ * \return nothing
+ */
```

Comment:
I overlooked that this already had a doxygen block with its header declaration, so we don't need a separate one here